api.planx.uk/helpers.ts

```diff
@@ -433,6 +434,9 @@ const dataMerged = async (
       ob[nodeId] = {
         type: ComponentType.InternalPortal,
         edges: [node.data?.flowId],
+        data: {
+          flattenedFromExternalPortal: true,
+        },
```

Comment:
These additions shouldn't have any functional change - just makes for easier auditability of what's happened on flatten ! Reminder that endpoint `/flows/:flowId/flatten-data` is straight-forward way to inspect this data structure